panasonic_camera/camera.py

```diff
@@ -93,6 +93,11 @@ def _validate_camrply(camrply: ET.Element) -> None:
     def _request(self, *args, **kwargs) -> ET.Element:
         kwargs.setdefault('timeout', 2)
         response = requests.get(self.cam_cgi_url, *args, **kwargs)
+        # If this request is trying to register with the camera,
+        # expect a simple text response, not XML. We'll put it
+        # in an XML tag anyway just to keep things tidy.
```

Comment:
I consider to introduce another (separate) request method that expects another response type. What is the full `response.text` that your `DC-FZ80` returns in case of `accctrl`? I guess, it is a comma separated list like the response of my HC-V380

```
err_param,V38061F1C5,remote,encrypted
```

The first element `err_param` seems to be the response code. The second element `V38061F1C5` seems to be the firmware version.

Reply:
Yes, good idea, we should have another request method when the comma separated list is expected.

My camera responds:
```
ok,FZ80-9D2C1C,remote,encrypted
```
9D2C1C could be the firmware version. In my camera UI when I check firmware version it says 1.1. (Also, weirdly, it says Copyright 2008 The Khronos Group Inc.)

FZ80-9D2C1C is also the SSID of the wifi network that the camera broadcasts.

Reply:
Done.